start

```diff
@@ -1,24 +1,53 @@
 #!/bin/bash
 
+local="true"
+withtemporal="false"
+
+while getopts lrt flag
+do
+    case "${flag}" in
+        l) local="true" ;;
+        r) local="false" ;;
+        t) withtemporal="true" ;;
+    esac
+done
+
 echo
 echo " * Bringing services up, (re)building as required..."
-echo
+echo "Local environment:" $local
+echo "With Temporal Server:" $withtemporal
+
+if [ $local == "true" ]; then
```

Comment:
I think this block would be more succinct if you build variables with strings like "" or "--profile withtemporal" and then interpolate those into a single `docker compose ...` command at the end.